packages/frontend-app-api/src/wiring/parameters.ts

```diff
@@ -19,6 +19,16 @@ import {
   Extension,
   ExtensionInstanceParameters,
 } from '@backstage/frontend-plugin-api';
+import { JsonValue } from '@backstage/types';
+import omitBy from 'lodash/omitBy';
+
+const knownExtensionInstanceParameters = [
+  'id',
```

Comment:
Nope!

Reply:
aw shucks! good catch, removed and updated test